lib/lte_link_control/lte_lc.c

```diff
@@ -139,7 +139,7 @@ K_WORK_DEFINE(lte_lc_edrx_ptw_send_work, lte_lc_edrx_ptw_send_work_fn);
 static void lte_lc_edrx_req_work_fn(struct k_work *work_item);
 K_WORK_DEFINE(lte_lc_edrx_req_work, lte_lc_edrx_req_work_fn);
 
-K_THREAD_STACK_DEFINE(lte_lc_work_q_stack, 1024);
+K_THREAD_STACK_DEFINE(lte_lc_work_q_stack, 768);
```

Comment:
744 of the stack is used when executing the code, i.e., sending AT%XMONITOR and dispatching the event.
Should we leave more space and actually not change the stack size?

Reply:
Let's leave this as is to have some room for changes in the code.